### DISCUSSION TITLE
Reboot of opencore installer with version 0.8.5

### DISCUSSION THREAD

**Discussion by andre1210** [simprecicchiani/ThinkPad-T460s-macOS-OpenCore#179] [Q&A]
Hi, I'm quite new to OpenCore. 
I have a t460s i7 6600U.
Installing Monterey with EFI folder of ThinkPad-T460s-macOS-OpenCore version 0.8.0 works without issues.
When I try to use the EFI folder of version 0.8.5, booting from the usb stick leads always to reboots.
I only see for a very short time:
OCS: No schema fro LegacyEnable at 2 index, content NVRAM!
The only change in the Config.plist is the PlatformInfo part (serial numbers and UUID).
I don't see any further errors.
How can I analyze the root cause?
I would be happy for any hints.
thank you
André

**Comment by alexdelli** [comment]
Hi I've the same isseue. Did you solve anyway?
thank you

**Comment by alexdelli** [comment]
Solved installing Ventura
